pkg/common/utils/utils.go

```diff
@@ -26,6 +28,8 @@ import (
 	"sigs.k8s.io/vsphere-csi-driver/pkg/csi/service/logger"
 )
 
+const DefaultQuerySnapshotLimit = int64(128)
```

Comment:
https://opengrok.eng.vmware.com/source/xref/main.perforce.1666/bora/vsan/pyvmodl/CnsVolumeManager.py#1953

```
   1953    Default cursor size is 128 currently.
```

There is no difference in the DefaultQuerySnapshotLimit, but importing the constant file is causing circular dependency. Ideally, the constants file should have been in a top-level package. I re-initialized the variable here to avoid the circular dependency. 
Due to an existing bug (bug 2813244) we will always set the limit from csi perspective.

Reply:
Looks like we should be re-using that one defined in constants.go.

Reply:
I actually can't use it since it creates an import cycle and fails in the compilation. To be able to use the constant file across multiple files without creating import cycles we need to refactor and move `pkg/csi/service/common/constants.go` to `pkg/constants/constants.go` , we faced a similar issue in our plugin code, and you'll notice that the constant file was moved. This is out of scope for this change, I can do it later.

Reply:
Please think about how to resolve this cyclic import issue. We can do it in separate PR. But it is not a good idea to keep two different constants for the same purpose.